content/en/data_jobs/dataproc.md

```diff
@@ -0,0 +1,107 @@
+   - Under **Secret value**, paste your Datadog API key in the **Secret value** text box.
+      {{< img src="data_jobs/dataproc/key_value.png" alt="GCP Secret Manager, Create secret. A section titled 'Secret details'. On the top, a text box containing 'datatdog_dd_api_key'. On the bottom, a text box to paste your own API key." style="width:80%;" >}}
+   - Then, click **Create Secret**.
+1. Under **Rotation** page, you can optionally turn on [automatic rotation][3].
+1. In [GCP Secret Manager][2], open the secret you created. Take note of the Resource ID, which is in the format "projects/<PROJECT_NAME>/secrets/<SECRET_NAME>".
+
+### Create and configure your Dataproc cluster
+
+When you create a new **Dataproc Cluster on Compute Engine** in the [Google Cloud Console][4], add an initialization action on the **Customize cluster** page:
+
+1. Save the following script to a GCS bucket that your Dataproc cluster can read. Take note of the path to this script.
+
+   ```shell
+   #!/bin/bash
+
+   # Set required parameter DD_SITE
+   DD_SITE={{< region-param key="dd_site" code="true" >}}
+
+   # Set required parameter DD_API_KEY with Datadog API key.
+   # The commands below assumes the API key is stored in GCP Secret Manager, with the secret name as datadog_dd_api_key and the project <PROJECT_NAME>.
+   # IMPORTANT: Modify if you choose to manage and retrieve your secret differently.
+   # Change the project name, which you can find on the secrets page. The resource ID is in the format "projects/<PROJECT_NAME>/secrets/<SECRET_NAME>".
+   gcloud config set project $PROJECT_NAME
+   SECRET_NAME=datadog_dd_api_key
+   DD_API_KEY=$(gcloud secrets versions access latest --secret $SECRET_NAME)
+
+   # Optional parameters
+   # Uncomment the following line to allow adding init script logs when reporting a failure back to Datadog. A failure is reported when the init script fails to start the Datadog Agent successfully.
+   # export DD_DJM_ADD_LOGS_TO_FAILURE_REPORT=true
+
+   # Download and run the latest init script
+   DD_SITE=$DD_SITE DD_API_KEY=$DD_API_KEY bash -c "$(curl -L https://dd-data-jobs-monitoring-setup.s3.amazonaws.com/scripts/dataproc/dataproc_init_latest.sh) || true
+
+   ```
+
+   The script above sets the required parameters, and downloads and runs the latest init script for Data Jobs Monitoring in Dataproc. If you want to pin your script to a specific version, you can replace the file name in the URL with `dataproc_init_<version_tag>.sh`, such as `dataproc_init_1.5.0.sh` to use the specific version you want.
+
+1. On the **Customize cluster** page, find the **Initialization actions** section. Click **Browse** to bring up the **Select object** dialog.
+   {{< img src="data_jobs/dataproc/browse-buckets.png" alt="Google Cloud, Dataproc, Create a Dataproc cluster on Compute Engine, Add Initialization Action dialog. Text fields for browsing buckets" style="width:80%;" >}}
```

Comment:
I would simplify a bit this part to something like this

```
On the Customize cluster page, locate the Initialization Actions section. Enter the path where you saved the script from the previous step
```

and change the screenshot to show the `Initialization actions` part

![image](https://github.com/user-attachments/assets/8803ecf1-f612-49e3-9cd9-4bbbf1c114d0)

Reply:
Thanks, addressed !